resources/views/maps/layers/_form.blade.php

```diff
@@ -25,6 +25,9 @@
 
 @php
     $options = $map->layerPositionOptions();
+    if (!empty($model->position) && $model->position == array_key_last($options) - 1) {
```

Comment:
Same here, move the logic to the function that generates the list, rather than adding the logic in the views :)